maquetado/index.html

```diff
@@ -9,6 +9,11 @@
   <link rel="stylesheet" type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1> 
+  <div class="react-tech">
+    <img src="assets/react_icon.svg" alt="react icon" class="react-icon">
+    <h1 class="title">ReactJS</h1> 
+    <h2 class="title-relase">Release: 2013</h2>
+    <h3 class="title-description">React is a declarative component-based JavaScript library for building user interfaces.</h3>
```

Comment:
This isn't a title. I recommend you this [article](https://medium.com/wolox-driving-innovation/what-margin-padding-taught-me-after-50-projects-6878b53c903f)